src/components/masters/program/ProgramPage.jsx

```diff
@@ -137,12 +143,22 @@ ProgramPage.propTypes = {
     programSlug: PropTypes.string.isRequired,
     programUUID: PropTypes.string.isRequired,
     programDocuments: PropTypes.shape({
-      display: PropTypes.bool,
-      header: PropTypes.string,
-      documents: PropTypes.arrayOf(PropTypes.shape({
-        display_text: PropTypes.string,
-        document: PropTypes.string,
-      })),
+      cover_image: PropTypes.string,
```

Comment:
You still have the branding schema inside the documents shape.

Reply:
😅 I guess I did it too fast... should be fixed now